app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(
+            self,
+            name: str,
+            age: int,
+    ) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict[str, str]]) -> list[Person]:
+    people_list = [
+        Person(person["name"], int(person["age"])) for person in people
+    ]
+    for person_data in people:
+        if person_data.get("wife"):
+            Person.people[person_data.get("name")].wife\
```

Comment:
There is no need to use the get method on lines 20 and 21. On line 19, you have already ensured that 'wife' exists in 'person_data'. Simplify it to just person_data["name"] and person_data["wife"]